### DISCUSSION TITLE
Keep track of a changelog on the website

### DISCUSSION THREAD

**Discussion by TiBiBa** [hedyorg/hedy#3620] [Website]
Quite often we implement new features, add elements, change the structure or do a complete overhaul of the UI. I can understand that this feels confusing for the end-user. What does the new button do? Why is the adventure window changed? To introduce a bit of explanation it might be nice to keep track of a changelog on the website. Where we enter changes that are relevant for the UX with the corresponding deploy date / time. For example,

- Button added to code page to enable user to hide / show the code editor (10 November)
- Profile page re-structured to be more responsive (06 November)

Not sure if this would be the ideal structure but having something of a log for changes relevant for the end user might be worth investing our time.

**Comment by Felienne** [comment]
What a great idea @TiBiBa!! In principle we could generate this from the PRs merged, a bit like hedy.org/version but with a better layout.

Maybe this is something we can build upon: https://dev.to/github/how-to-automatically-generate-release-notes-for-your-project-2ng8

Of course that commits back to Github and we'd like something user facing. 

Do you want to explore this further yourself?
